jetstream/tests.py

```diff
@@ -29,20 +30,33 @@ class TestJetstream(TestCase):
     def setUp(self):
         pass
 
+    def _check_cassette(self, cassette, expected_cassette_length):
+        """Make sure the cassette is:
+         1. The expected length and either
+         2. New (dirty) or
+         3. Existing, rewound
+         """
+        self.assertEqual(len(cassette), expected_cassette_length)
+        self.assertTrue(cassette.dirty or cassette.all_played)
```

Comment:
I have two nitpicks about this method. I would name it `assert_cassette_playback_length`. When I first saw `check_cassette` i had to go look it up, because it wasn't clear (why does it require a number as an argument?). The other is that this method should exist outside of this test. It could also be a decorator, since it is something we would apply to most cassette tests. 

major 👍  for working on tests.